src/main/java/de/dhbw/karlsruhe/adapters/cli/output/LogoutCliAdapter.java

```diff
@@ -0,0 +1,26 @@
+package de.dhbw.karlsruhe.adapters.cli.output;
+
+import de.dhbw.karlsruhe.adapters.CliOutputPort;
+import de.dhbw.karlsruhe.domain.ports.dialogs.LogoutCliPort;
+import de.dhbw.karlsruhe.domain.services.DependencyFactory;
+
+public class LogoutCliAdapter implements LogoutCliPort {
+
+    private final CliOutputPort cliOutputPort = DependencyFactory.getInstance().getDependency(CliOutputPort.class);
```

Comment:
In den anderen Adaptern nutzt du an dieser Stelle den CliOutputAdapter, gibt es einen Grund warum es hier besser ist den Port zu nehmen?

Reply:
Nein, sollte immer der Port sein. Habe ich übersehen. Danke!